packages/istanbul-lib-source-maps/test/map-store.test.js

```diff
@@ -79,8 +79,8 @@ describe('map store', function () {
             "column": 0
           },
           "end": {
-            "line": null,
-            "column": -1
+            "line": 1,
+            "column": 13
```

Comment:
It seemed a bit weird that the `decl` mapping for `(anonymous_1)` suddenly started mapping to something; what is `(anonymous_1)`? I think this might be referring to `export class` as if it represents a function?

Reply:
I'm having a hard time figuring out how this data blob works, so not sure I can add anything.

Reply:
@loganfsmyth is it possible we're treating the `export` or `class` snippet as a `function` token type in the AST, and this is what `(anonymous_1)` refers to? The new offsets seem better than `null`, and `-1` ... which seems like a bug with a test around it, no?

@coreyfarrell does this set off any alarm bells for you?

Reply:
The `(anonymous_1)` is referring to the IIFE.

So we've instrumented the transpiled source rather than instrumenting the original source.  I'm not fully sure how this is supposed to work, on my own projects I put instrumentation before other babel transformations.  I realize that could probably be a problem with TypeScript (the code added by instrumentation may not pass TypeScript checks).